src/Microsoft.DncEng.SecretManager/SecretTypes/ZenHubAccessToken.cs

```diff
@@ -0,0 +1,37 @@
+using System;
```

Comment:
header needed

Reply:
most files in this repo don't have the headers. I'd rather add them all in one separate PR, than just add a few here.

Reply:
Yeah, I think a lot of this was me.  I should be able to have ReSharper do it all in one fell swoop, I think...  should I make a PR for that? Should be a fun merge conflict.